podman_build.sh

```diff
@@ -1,22 +1,30 @@
 #!/bin/bash
 
 if [[ "$1" == "" ]]; then
-    IMAGES_TO_BUILD=("backend", "scheduler", "worker", "flower", "frontend")
+    IMAGES_TO_BUILD=("backend" "scheduler" "worker" "flower" "frontend")
 else
     IMAGES_TO_BUILD=("$1")
 fi
 
 # Image info
 IMAGE_TAG=$(git rev-parse --short=7 HEAD)
-IMAGE_PREFIX="quay.io/cloudservices/ibutsu-"
+IMAGE_PREFIX="ibutsu/"
```

Comment:
I assume this script has been made for cases when we want to build image locally and push it to quay registry. So I would keep correct prefix - `IMAGE_PREFIX="quay.io/ibutsu/"`

Reply:
yeah, I think I'm going to move this to a flag or arg, because I'd like the default here to just build local images aligned with the image names in ibutsu_pod.yml and/or the docker-compose file.

That way it can be used to build the images locally without tagging them with the upstream destination.

I can set the default to be quay.io tagged, so when building just for local use the flag gets passed?